packages/swapper/src/swappers/thorchain/utils/bitcoin/utils/getThorTxData.ts

```diff
@@ -55,26 +58,25 @@ export const getThorTxInfo: GetBtcThorTxInfo = async ({
         details: { inboundAddresses }
       })
 
-    const priceRatio = await getPriceRatio(deps, {
-      buyAssetId: buyAsset.assetId,
-      sellAssetId: sellAsset.assetId
-    })
-
-    const expectedBuyAmount = toBaseUnit(
-      fromBaseUnit(bnOrZero(sellAmount).dividedBy(priceRatio), sellAsset.precision),
-      buyAsset.precision
+    const tradeRate = await getTradeRate(sellAsset, buyAsset.assetId, sellAmount, deps)
```

Comment:
stylistic/maintainability comment (not a requested change) - passing an object rather than higher arity functions is generally a little easier to maintain/convention around the rest of this code.

Reply:
I agree and I need to get in that habit. will try to write functions that way from now on